### ISSUE TITLE
Debugger not working

### ISSUE THREAD

**Issue by atirutw** [zenith391/OCEmu#37]
I used the version from AUR in combination with Lua 5.3(I edited the `/usr/bin/ocemu` for this).

```
$ ocemu --basedir=/home/atirut/Projects/ocemu-machine --debugger
false   apis/debugger.lua:189: attempt to index a nil value (field '?')
stack traceback:
        apis/debugger.lua:189: in upvalue 'drawTabs'
        apis/debugger.lua:195: in upvalue 'drawUI'
        apis/debugger.lua:199: in field '?'
        boot.lua:315: in function <boot.lua:77>
        [C]: in function 'xpcall'
        boot.lua:327: in main chunk
        [C]: in ?
```

**Comment by zenith391** [comment]
Yea sorry, it's because the only implemented tab is the sound card (although I'm prob gonna add more).

**Comment by zenith391** [comment]
closed by [3db4e19](https://github.com/zenith391/OCEmu/commit/3db4e19f54f04298e66d6bedb82ad8921d4794ac)

**Comment by zenith391** [comment]
You can see currently implemented tabs on the [wiki page](https://github.com/zenith391/OCEmu/wiki/Debugger#debugger-tabs).
